aws/account-settings/variables.tf

```diff
@@ -1,3 +1,8 @@
+variable "minimum_password_length" {
+  type        = "string"
+  description = "Minimum number of characters allowed in an IAM user password.  Integer between 6 and 128, per https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_passwords_account-policy.html"
```

Comment:
@osterman :: does this new variable need a `default` value?

I'm thinking that committing it "as-is" will break upstream users by requiring a new variable and value declaration.

Reply:
I would prefer to set a sane default

Reply:
module we call currently has a default of `8`:
https://github.com/cloudposse/terraform-aws-iam-account-settings/pull/1/files#diff-8e2475e53e84b1e038e9f5d670c3faefR12

Reply:
Fixed in https://github.com/cloudposse/terraform-root-modules/pull/121/commits/2292b1c1cab61817dfae8a2f502aaa9c483e5516